File: app/src/main/java/podtest/jiki/se/podtest/model/Pagination.java
package podtest.jiki.se.podtest.model;

public class Pagination {

    private Integer page;
    private Integer size;
    private Integer totalhits;
    private Integer totalpages;
    private String nextpage;

    public Integer getPage() {
        return page;
    }

    public Integer getSize() {
        return size;
    }

    public Integer getTotalhits() {
        return totalhits;
    }

    public Integer getTotalpages() {
        return totalpages;
    }


    public String getNextpage() {
        return nextpage;
    }


}

File: app/src/main/java/se/asapehrsson/podtest/data/Pagination.kt
package se.asapehrsson.podtest.data

import com.google.gson.annotations.SerializedName

data class Pagination(
        @SerializedName("page") val page: Int? = null,
        @SerializedName("size") val size: Int? = null,
        @SerializedName("totalhits") val totalhits: Int? = null,
        @SerializedName("totalpages") val totalpages: Int? = null,
        @SerializedName("nextpage") val nextpage: String? = null
)
